web/templates/BluebookPerUser.tpl

```diff
@@ -242,6 +242,7 @@ for compressibility purposes by tools/build.php
 					<span class="btn-group">
 						<button class="btn list-table-btn hide"><i class="icon-th"></i></button>
 						<button class="btn download-csv-btn hide"><i class="icon-download-alt"></i></button>
+            <button class="btn download-ics-btn hide"><i class="icon-calendar"></i></button>
```

Comment:
Given that the ICS generation only works for the 202001 season, would it possible to hide/disable the button when on other seasons?

Reply:
I'm thinking of doing this in the worksheets.js file. Is there an advantage to doing it here?

Reply:
You're right, it's probably better to do it there.

Reply:
So, I did a little experimentation. If we toggle the ics button in BluebookPerUser.tpl, we need to refresh the page in order to see only two buttons. But if in worksheets.js, it's more dynamic but when we see only two buttons, the right edges of the csv download button aren't rounded. Combining the two results in the same effect as having the season check in BluebookPerUser.tpl. This assumes, of course, I correctly implemented the above. What are your thoughts?

Reply:
This approach seems better than putting it in BluebookPerUser.tpl. 

I don't think the rounding is a big issue. It might be fixed by putting the calendar button in the middle of the three, but honestly not worth fixing.